woocommerce/woocommerce-ios
# Order Creation: Update feedback banner on Orders tab for Order Creation & Simple Payments

Closes: #6177
⚠️ Depends on #6537 ⚠️ 

## Description

This updates the feedback banner on the Orders tab to include details for both Order Creation and Simple Payments. It also updates the survey linked in the banner, to be able to distinguish feedback for each of those features.

## Changes

* Removes `SimplePaymentsTopBannerFactory` and adds `OrdersTopBannerFactory` to create the updated feedback banner for the Orders tab.
* Removes the simple payments banner and adds the updated orders banner as the default banner (when there are no errors) in `OrderListViewModel` and `OrderListViewController`.
* Removes the simple payments prototype survey and adds the new orders survey (testing and production variations) in `WooConstants`, `WooAnalyticsEvent`, and `SurveyViewController`.
* Updates unit tests.

## Testing

1. Build and run the app.
2. Go to the Orders tab and confirm the updated banner is displayed (new title and info text).
3. Select "Give feedback" and confirm the new survey is displayed.

## Screenshots

Orders banner|Feedback survey
-|-
![Simulator Screen Shot - iPhone 13 Pro - 2022-03-29 at 17 28 29](https://user-images.githubusercontent.com/8658164/160668354-8137d92f-a33d-41c4-8edb-8b3e4ce3288f.png)|![Simulator Screen Shot - iPhone 13 Pro - 2022-03-29 at 17 28 35](https://user-images.githubusercontent.com/8658164/160668371-96ad0a27-9fb3-428b-957b-91366aa8b23f.png)

## Submitter Checklist

Update release notes:
